PDO/PDO.php

```diff
@@ -1756,8 +1764,8 @@ public function fetchColumn(#[LanguageLevelTypeAware(['8.0' => 'int'], default:
         #[TentativeType]
         public function fetchAll(
             #[LanguageLevelTypeAware(['8.0' => 'int'], default: '')] $mode = PDO::FETCH_DEFAULT,
-            #[PhpStormStubsElementAvailable(from: '5.3', to: '7.4')] $fetch_argument = null,
-            #[LanguageLevelTypeAware(['8.0' => 'mixed'], default: '')] ...$args
+            #[LanguageLevelTypeAware(['8.0' => 'int|string|callable|null'], default: '')] $fetch_argument = null,
```

Comment:
Since the doc say `They can't be used with named parameters.`
https://www.php.net/manual/en/pdostatement.fetchall.php#refsect1-pdostatement.fetchall-parameters

The name doesn't really matter for the second param, so I merged it with the 7.4 signature.